.drone.yml

```diff
@@ -62,1651 +62,1651 @@ trigger:
   event:
     exclude:
     - promote
+# ---
```

Comment:
is this still supposed to be commented out?

Reply:
removed, was added for v2 prov tests